app/controllers/stripe-setup/stripe-setup-link/stripe_setup_dashboard_redirect_controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const paths = require('../../../../app/paths')
+const { ConnectorClient } = require('../../../services/clients/connector_client')
+const connectorClient = new ConnectorClient(process.env.CONNECTOR_URL)
+const logger = require('../../../utils/logger')(__filename)
+const { keys } = require('@govuk-pay/pay-js-commons').logging
+
+function getTargetServiceForRedirect (user, externalServiceId) {
+  return user.serviceRoles.filter((serviceRole) => serviceRole.service.externalId === externalServiceId)[0]
+}
+
+module.exports = async (req, res) => {
+  const externalServiceId = req.params.externalServiceId
+  const targetServiceRoleForRedirect = getTargetServiceForRedirect(req.user, externalServiceId)
+  if (targetServiceRoleForRedirect) {
+    const result = await connectorClient.getAccounts(targetServiceRoleForRedirect.service.gatewayAccountIds)
```

Comment:
I think this function takes a `param` object so I don't think this would work, I think this means the happy path test will need to be adjusted. 

From what I can see the usage is `{ gatewayAccountIds: [ ] }`